app/controllers/lessons_controller.rb

```diff
@@ -18,4 +18,14 @@ def create
     end
   end
 
+  def show
+    @lesson = Lesson.find(params[:id])
+    @lesson_words = @lesson.lesson_words
+
+    @count = 0
+    @lesson_words.each do |l|
+      @count += 1 unless l.word_answer.correct == false
```

Comment:
@shokogyu You can create a method in `Lesson` model , that will get all correct answers.

Create relationship: `has_many :word_answers, through: :lesson_words`
So you can do create a method like this:
```
def correct_answers
  self.word_answers.where(is_correct: true)
end
```